src/shared/dissect-image.c

```diff
@@ -2040,7 +2040,7 @@ static int mount_point_is_available(const char *where, const char *path, bool mi
                 return false;
         if (r < 0)
                 return log_debug_errno(r, "Failed to check directory \"%s\": %m", p);
-        return true;
+        return r > 0;
```

Comment:
Whoa, where did the whole patchset go? No we're down to a one-bit fix? It really pays not to review things quickly ;)

Reply:
this PR was sponsored by Agatha Christie